SmartDeviceLink/SDLRdsData.h

```diff
@@ -0,0 +1,86 @@
+//
+//  SDLRdsData.h
+//
+
+#import "SDLRPCMessage.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SDLRdsData : SDLRPCStruct
+
+- (instancetype)initWithProgramService:(nullable NSString *)PS;
```

Comment:
Is only one item ever set in this struct at a time? If not, why not have all of the parameters at once. Also, spell out the parameter name.